dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -61,15 +65,58 @@ private static void printDependencyTree(String coordinate)
    * Prints dependencies expressed in dependency paths in tree in similar way to mvn
    * dependency:tree.
    *
-   * @param dependencyPaths sorted dependency paths
+   * @param dependencyPaths dependency paths from @{@link
+   *     DependencyGraphBuilder#getCompleteDependencies(Artifact)}. Each element must have its
+   *     parent in the list, except the ones at the root.
    */
   public static String formatDependencyPaths(List<DependencyPath> dependencyPaths) {
     StringBuilder stringBuilder = new StringBuilder();
+    // Printing text representing a tree requires traversing the items in pre-order
+    ListMultimap<DependencyPath, DependencyPath> tree = buildDependencyPathTree(dependencyPaths);
+    // Empty dependency path is to retrieve children of root node
+    formatDependencyPathTreePreOrder(stringBuilder, tree, new DependencyPath());
+    return stringBuilder.toString();
+  }
+
+  private static void formatDependencyPathTreePreOrder(
```

Comment:
wasn't "pre-order" renamed to "full level order" in other places?

Reply:
When printing dependency tree in text format, pre-order is easier.

As it's confusing, I renamed the function to formatDependencyPaths.